backend/chainlit/config.py

```diff
@@ -233,6 +237,7 @@ class ChainlitConfig:
     chainlit_prod_url = chainlit_prod_url
 
     run: RunSettings
+    features: FeaturesSettings
```

Comment:
I would name it `FeatureSettings` for consistency

Reply:
Gotcha, I may be a bit nitpicky but I think that the straightforward convention is a PascalCase "{{ header }}Settings" so in our case I would keep the plural. What do you think?